js/ui/search.js

```diff
@@ -497,6 +501,19 @@ var SearchResults = class {
         this._reloadRemoteProviders();
     }
 
+    _reloadInternetProviders() {
+        if (this._internetProvider)
+            this._unregisterProvider(this._internetProvider);
+        this._internetProvider = null;
```

Comment:
nitpick: this is not needed as you assign it again in the next line.

Reply:
Addressed.